packages/addon/lua/gelly/mods/blood-mod/init.lua

```diff
@@ -2,26 +2,47 @@
 print("Blood mod loaded")
 
 local BLOOD_CONFIGS = {
-	[DMG_BULLET] = {
-		MinDensity = 200,
+    {
+        DamageFlags = bit.bor(DMG_BULLET, 8194),
```

Comment:
Is there a reason for including the `DMG_ALWAYSGIB` here instead of just `DMG_BULLET`?

Also `8194` is the same as `bit.bor(DMG_BULLET, DMG_ALWAYSGIB)` which we should use here, and can remove the redundant bitwise or operation between that combined flag and `DMG_BULLET` again

Reply:
Ah yes i just really copied from what was before which was 8194 and DMG_BULLET good catch

Reply:
I think @yogwoggf left a comment about 8194 being bullets in m9k iirc?

Reply:
Unfortunately you’ve been mislead by @yogwoggf misunderstanding of how damage types work

I imagine he was confused as to why m9k weapons weren’t working, printed their damage flags (a combination of bullet and gib), and pasted the number he saw

Reply:
Thx